recipes/sirius/meta.yaml

```
@@ -0,0 +1,68 @@
+{% set name = "sirius" %}
+{% set version = "7.2.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/electronic-structure/SIRIUS/archive/refs/tags/v{{ version }}.tar.gz
+  sha256: 01bf6c9893ff471473e13351ca7fdc2ed6c1f4b1bb7afa151909ea7cd6fa0de7
+  patches:
+    - fix_osx_compilation.patch  # [osx]
+
+build:
+  number: 0
+  skip: True  # [win]
+  run_exports:
+    - {{ pin_compatible('sirius', max_pin='x') }}
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+    - {{ compiler('fortran') }}
+    - make
+    - cmake >=3.14.0
+  host:
+    - {{ mpi }}
+    - libblas
+    - liblapack
+    - gsl
+    - libxc
+    - spglib
+    - hdf5
+    - spfft
+    - spla
+    - libgomp      # [linux]
+    - llvm-openmp  # [osx]
```

According to https://conda-forge.org/docs/maintainer/knowledge_base.html#openmp these should be in `build`
```suggestion
    - libgomp      # [linux]
    - llvm-openmp  # [osx]
```

---

Thanks for catching that, I've moved them to build